app-config.yaml

```diff
@@ -211,7 +211,7 @@ integrations:
   #    clientEmail: 'example@example.com'
   #    privateKey: ${GCS_PRIVATE_KEY}
   awsS3:
-    - host: amazonaws.com
+    - endpoint: ${AWS_S3_ENDPOINT}
```

Comment:
There is no `host` property in the `aws3` config - https://github.com/backstage/backstage/blob/1a14adeb574f7846b6d5f5ef555a57dea403e03a/packages/integration/config.d.ts#L253 - replace with `endpoint`